src/main/java/github/grace000/erversay/RouteHandlers/NotAllowed.java

```diff
@@ -24,11 +23,12 @@ public boolean isMethodAllowed(String method) {
         return false;
     }
 
-    public String handle(Request request) {
+    public Response handle(Request request) {
         if (isMethodAllowed(request.method)) {
             return responseBuilder
                     .build();
-        } else return responseBuilder
+        }
+        else return responseBuilder
```

Comment:
Please also wrap `if` and `else` clauses in curly brackets. [Bad things](https://blog.codecentric.de/en/2014/02/curly-braces/) happen to those who don't. Also, you should consider using [guard statements](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) to return early when given bad input.